README.md

```diff
@@ -1,3 +1,110 @@
-# module-ballerinax-guidewire.insnow
+# Ballerina Guidewire InsuranceNow connector
 
-Ballerina Guidewire InsuranceNow REST API client connector module.
+[![Build](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/actions/workflows/ci.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/actions/workflows/ci.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/actions/workflows/trivy-scan.yml)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/actions/workflows/build-with-bal-test-native.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-guidewire.insnow.svg)](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/commits/master)
+[![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/github.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module%2Fguidewire.insnow)
+
+## Build from the source
+
+### Setting up the prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 17. You can download it from either of the following sources:
+
+    * [Oracle JDK](https://www.oracle.com/java/technologies/downloads/)
+    * [OpenJDK](https://adoptium.net/)
+
+   > **Note:** After installation, remember to set the `JAVA_HOME` environment variable to the directory where JDK was installed.
+
+2. Download and install [Ballerina Swan Lake](https://ballerina.io/).
+
+3. Download and install [Docker](https://www.docker.com/get-started).
+
+   > **Note**: Ensure that the Docker daemon is running before executing any tests.
+
+4. Export Github Personal access token with read package permissions as follows,
+
+    ```bash
+    export packageUser=<Username>
+    export packagePAT=<Personal access token>
+    ```
+
+## OpenAPI CLI command
```

Comment:
IMO we should make this a H3 title (to be placed under `Build from the source`).

Additionally, could we consider using a more contextually appropriate title, such as the one below?
```suggestion
### Generating the client using OpenAPI tool
```